client/my-sites/marketplace/components/billing-interval-switcher/index.tsx

```diff
@@ -57,7 +89,25 @@ const BillingIntervalSwitcher: React.FunctionComponent< Props > = ( {
 	return (
 		<Container>
 			<PlansIntervalToggleLabel>{ translate( 'Price' ) }</PlansIntervalToggleLabel>
-			<PlansIntervalToggle intervalType={ billingPeriod } onChange={ onChange } />
+			<PlansIntervalToggle intervalType={ billingPeriod } onChange={ onChange }>
+				{ plugin ? (
+					<PluginAnnualSaving plugin={ plugin }>
+						{ ( annualSaving: { saving: any } ) =>
+							annualSaving.saving ? (
+								<>
+									<PluginAnnualSavingLabelDesktop>
+										(
```

Comment:
These wrapping parens can probably be omitted

Reply:
Removed in [this commit](https://github.com/Automattic/wp-calypso/pull/61341/commits/35806f3d082ede6f7e5577351afd86440f1334d6).